### PR TITLE
handle localization of plurals

### PR DESCRIPTION
**Pull request by JessicaBarh**

### CODE REVIEW

File: src/commandsAndMenu.tsx [hunk 1]
```diff
@@ -716,11 +716,11 @@ export function addCommands(
       const extensions = files
         .map(file => PathExt.extname(file.to))
         .filter(extension => extension.length > 0);
-      const subject = extensions.length > 1 ? 'extensions' : 'extension';
-      return trans.__(
-        'Ignore %1 %2 (add to .gitignore)',
-        extensions.join(', '),
-        trans.__(subject)
+      return trans._n(
+        'Ignore %2 extension (add to .gitignore)',
```
Comment (goanpeca) [inline src/commandsAndMenu.tsx:720]: This should probably be a `%1` ?

File: src/commandsAndMenu.tsx [hunk 2]
```diff
@@ -716,11 +716,11 @@ export function addCommands(
...
+        'Ignore %2 extensions (add to .gitignore)',
```
Comment (goanpeca) [inline src/commandsAndMenu.tsx:721]: This should probably be a `%1` ?

**Review by goanpeca — CHANGES_REQUESTED** [review]
Thanks @JessicaBarh !

File: src/commandsAndMenu.tsx (hunk 2 above)
Comment (JessicaBarh) [reply src/commandsAndMenu.tsx:721]: using %1 was actually picking up the number arg (extensions.length). I was testing with this: https://github.com/JessicaBarh/jupyterlab-fr_FR/tree/handle-plurals/jupyterlab_language_pack_fr_FR/extensions/jupyterlab-git

Comment (goanpeca) [reply src/commandsAndMenu.tsx:721]: You are absolutely right! My bad

**Review by fcollonval — APPROVED** [review]
Thanks for the follow-up!

**Pull request merged**